src/gcode-machine-control_test.cc

```diff
@@ -110,8 +118,8 @@ class Harness {
     return machine_control->ParseEventReceiver();
   }
 
-  MockMotorOps expect_motor_ops_;
-  HardwareMapping hardware_;
+  MockMotorOps expect_motor_ops;
+  HardwareMapping hardware;
```

Comment:
we never access hardware, so we should be explicit and make it private (and keep the underscore), and put `machine_control` and `expect_motor_ops` in the public.

Reply:
(solved)